GVRf/Framework/jni/shaders/material/unlit_shader.cpp

```diff
@@ -84,6 +84,11 @@ void UnlitShader::render(const glm::mat4& mvp_matrix,
         std::string error = "UnlitShader::render : texture with wrong target";
         throw error;
     }
+    if (texture->getId() == 0)
+    {
+        std::string error = "UnlitShader::render : texture with invalid Id";
+        throw error;
```

Comment:
As a note,
the if (gl_texture_ == 0)  block is not a debugging code but an error check routine to avoid app crash. Along with “throw error”, it will be consistent to egvrf’s error handling convention.